packages/docs/stories/templates/appshell.stories.tsx

```diff
@@ -0,0 +1,503 @@
+      align-items: center;
+      border: 1px dashed #9747FF;
+      border-radius: var(--syn-border-radius-medium);
+      color: #9747FF;
+      display: flex;
+      flex: 1 0 auto;
+      flex-direction: column;
+      font: var(--syn-body-small-bold);
+      justify-content: center;
+      margin: var(--syn-spacing-2x-large) var(--syn-spacing-2x-large) 0;
+    }
+
+    .synergy-demo-main h1 {
+      color: var(--syn-typography-color-text);
+      font: var(--syn-heading-large);
+      margin: 0;
+    }
+
+    .synergy-demo-footer > nav {
+      align-items: center;
+      display: flex;
+      font: var(--syn-body-x-small-regular);
+      gap: var(--syn-spacing-large);
+      height: var(--syn-spacing-3x-large);
+      justify-content: center;
+    }
+
+    .synergy-demo-footer :is(a, strong) {
+      color: var(--syn-input-help-text-color);
+    }
+
+    .synergy-demo-footer a:hover {
+      color: var(--syn-color-primary-600);
+    }
+
+    .synergy-demo-footer a:active {
+      color: var(--syn-color-primary-700);
+    }
+  </style>
+`;
+
+/**
+ * Create the defaut syn-prio-nav for all stories
+ */
+const createPrioNav = () => html`
+  <!-- prio-nav -->
+  <syn-prio-nav slot="navigation">
+    <syn-nav-item current horizontal>${getTranslation('appShell.navigation.home')}</syn-nav-item>
+    <syn-nav-item horizontal>${getTranslation('appShell.navigation.documents')}</syn-nav-item>
+    <syn-nav-item horizontal>${getTranslation('appShell.navigation.applications')}</syn-nav-item>
+    <syn-nav-item horizontal>${getTranslation('appShell.navigation.teams')}</syn-nav-item>
+  </syn-prio-nav>
+  <!-- /prio-nav -->
+`;
+
+/**
+ * Create a header with optional prio-nav
+ */
+const createHeader = ({
+  withMetaNavigation = false,
+} = {}) => html`
+  <!-- header -->
+  <syn-header>
+    ${getTranslation('appShell.appName')}
+    
+    ${withMetaNavigation ? createPrioNav() : ''}
+
+    <!-- meta-navigation -->
+    <nav slot="meta-navigation">
+      <syn-dropdown>
+        <syn-icon-button
+          color="neutral"
+          name="more_vert"
+          label="More"
+          slot="trigger"
+        ></syn-icon-button>
+        <syn-menu>
+          <syn-menu-item>${getTranslation('appShell.metaNavigation.menuItem')}</syn-menu-item>
+          <syn-menu-item>${getTranslation('appShell.metaNavigation.menuItem')}</syn-menu-item>
+          <syn-menu-item>${getTranslation('appShell.metaNavigation.menuItem')}</syn-menu-item>
+          <syn-menu-item>${getTranslation('appShell.metaNavigation.menuItem')}</syn-menu-item>
+          <syn-menu-item>${getTranslation('appShell.metaNavigation.menuItem')}</syn-menu-item>
+        </syn-menu>
+      </syn-dropdown>
+    </nav>
+    <!-- /meta-navigation -->
+  </syn-header>
+  <!-- /header -->
+`;
+
+/**
+ * Create a side navigation with an optional rail mode
+ */
+const createSideNav = ({
+  open = false,
+  rail = false,
+} = {}) => html`
+  <!-- side-navigation -->
+  <syn-side-nav
+    no-focus-trapping
```

Comment:
This shouldn't be added to every example! IT shouldn't be ther ee. g. for the first one.

Reply:
In addition @kirchsuSICKAG  we have a problem here: The header should be reachable, as it's not affected by the overlay. Otherweise you can't reach the close button anymore. We should provide a fix for that and write a new issue.

Reply:
The reason for this is that we do not use iframes on the the docs page. When using the iFrame approach, dark mode would not work. Will have to check if it is possible in another way, but having "global" components as the shell or dialog without killing the focus trapping will lead to strange behavior when navigating the docs overview pages.

Reply:
Easiest solution would be to remove the prop and set the items into iFrames. However, some stuff (like dark mode) may not work in this case. Like I said, I will check this. Its unfortunate that stuff like settings attributes or other plugins do not work in the docs overview when using "real" isolation (yet) :(. It also leads to bleeding, which is why I would love to have the iframe approach everywhere. This is also why I prefer the story approach instead of the overview, but this seems to be personal taste :)

Reply:
You are right, the focus handling is kind of strange currently. There seems to be a hidden element that is focused when using the navigation. Also, it does not honor the close icon in the header. Will create a ticket for this. It does seem to be related to the sidebar/header components as it also appears to be a problem in the default sidebar stories.

The main problem (adding `no-focus-trapping` on everything) is now fixed.

Reply:
The problem is caused by [shoelaces drawer implementation](https://github.com/shoelace-style/shoelace/blob/next/src/components/drawer/drawer.component.ts#L342) that adds the panel as focusable element. The same can be seen at the Shoelace demos of `<sl-drawer>`. Will open a ticket and see what we can do there.

Reply:
https://github.com/shoelace-style/shoelace/issues/2018